### PR TITLE
Add chef 15/16 support

### PR DESCRIPTION
**Pull request by nvwls**
Cosmetic changes to match cookstyle

### CODE REVIEW

**Comment by nvwls** [comment]
I updated the resources to work with chef-16

**Comment by richardlock** [comment]
Hi Joe, thanks for this.  We are moving away from Chef here, so would be happy to transfer ownership of this cookbook to the sous-chefs or somewhere else that's appropriate?

**Comment by nvwls** [comment]
Yes, sous-chefs would be appropriate.

**Comment by xorima** [comment]
Hey, 

Sous-Chefs would happily have this cookbook (And any others you want to offload), as long as you have no issues transferring it over to us? 

Thanks!
Jason 
sous-chefs board member

**Comment by richardlock** [comment]
No problem.  I think Dan owns it in Supermarket, but the repo is still hosted on our GitHub.  Let me know if you need any input to transfer it.  Thanks, Rich

**Comment by xorima** [comment]
If you can transfer the repo(s) I can get supermarket migrated over. (Always surprises me how much @damacus owns :D)

Did you also want to transfer over all your other application style cookbooks if you are not going to be using them in future, so at least we can look after the code base and ensure they can always get released etc.

**Comment by damacus** [comment]
Thanks @richardlock and @rbarkley, this one is a blast from the past.

If either of you want to make any further contributions to the cookbook, feel free to join Sous

**Comment by richardlock** [comment]
Hi Dan, thanks.

**Comment by richardlock** [comment]
@Xorima - We still use the PeopleSoft/Oracle cookbooks for upgrading legacy apps, but I'm happy to transfer anything else which might be useful to people.

File: metadata.rb [hunk 1]
```diff
@@ -10,4 +10,4 @@
 supports 'redhat'
 
 depends 'compat_resource', '~> 12.14'
```
Comment (tas50) [inline metadata.rb:12]: This really needs to go.

**Comment by xorima** [comment]
Can you transfer this repo over to us so we can start our update works please?

**Comment by richardlock** [comment]
Hi @Xorima, I get error "You don’t have the permission to create public repositories on sous-chefs" when I try to transfer the repo to sous-chefs.

**Comment by xorima** [comment]
Ah that will be due to you not being a member of sous-chefs, you can either transfer it to me and I will then move it in, or you can become a member of sous-chefs here: https://github.com/sous-chefs/terraform-github-org

**Comment by xorima** [comment]
And done, thank you very much @richardlock  and @nvwls for getting this transferred over to us.

**Pull request merged**